content/en/_index.md

```diff
@@ -28,36 +29,26 @@ Kubernetes.
-update, removal, and configuration of shims, reducing manual errors and improving reliability in
-managing WebAssembly (Wasm) workloads and other containerd extensions.
-
-**Spin Plugin for Kubernetes** The [Spin plugin for
-Kubernetes](https://github.com/spinkube/spin-plugin-kube) is designed to enhance Kubernetes by
-enabling the execution of Wasm modules directly within a Kubernetes cluster. This plugin works by
-integrating with containerd shims, allowing Kubernetes to manage and run Wasm workloads in a way
-similar to traditional container workloads.
-
-**Spin Operator** The [Spin Operator](https://github.com/spinkube/spin-operator/) enables deploying
-Spin applications to Kubernetes. The foundation of this project is built using the
-[kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework. Spin Operator defines Spin
-App Custom Resource Definitions (CRDs). Spin Operator watches SpinApp Custom Resources e.g. Spin app
-image, replicas, schedulers and other user-defined values and realizes the desired state in the
-Kubernetes cluster. Spin Operator introduces a host of functionality such as resource-based scaling
+[**Containerd Shim Spin**](https://github.com/spinkube/containerd-shim-spin) provides a shim for running Spin
+workloads managed by containerd. The Spin workload uses this shim as a runtime class within Kubernetes enabling
+these workloads to function similarly to container workloads in Pods in Kubernetes.
+
+[**Runtime Class Manager**](https://github.com/spinkube/runtime-class-manager) is an operator that
+automates and manages the lifecycle of containerd shims in a Kubernetes environment. This includes tasks
+like installation, update, removal, and configuration of shims, reducing manual errors and improving
+reliability in managing WebAssembly (Wasm) workloads and other containerd extensions.
+
+[**Spin Operator**](https://github.com/spinkube/spin-operator/) is a Kubernetes operator that enables
+deploying and running Spin applications in Kubernetes. It houses the SpinApp and SpinAppExecutor CRDs
+which are used for configuring the individual workloads and workload execution configuration such as
+runtime class. Spin Operator introduces a host of functionality such as resource-based scaling,
 event-driven scaling and much more.
 
-**Spin Trigger MQTT** [Spin Trigger MQTT](https://github.com/spinkube/spin-trigger-mqtt/) is an MQTT
-trigger designed specifically for Spin. It enables seamless integration between Spin and MQTT-based
-systems, allowing you to automate workflows and trigger actions based on MQTT messages.
+[**Spin kube plugin**](https://github.com/spinkube/spin-plugin-kube) is a plugin for the [Spin](https://developer.fermyon.com/spin/v3/index) CLI
+thats aim to ease the experience for scaffolding, deploying and inspecting Spin workloads in Kubernetes.
+
+[**Spin Trigger MQTT**](https://github.com/spinkube/spin-trigger-mqtt/) trigger enables seamless integration
```

Comment:
I personally find it confusing to list this trigger since there are more triggers than this one that the shim supports. Yes, this trigger lives in the SK org but i dont think we need to document it here